docs/conf.py

```diff
@@ -0,0 +1,110 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+# sys.path.insert(0,os.path.dirname(sys.path[0]))
+
+# -- Project information -----------------------------------------------------
+
+project = "WaterTAP-SETO"
+copyright = "2022-2023, NREL et al."
+author = "WaterTAP-SETO dev team"
+
+# The full version, including alpha/beta/rc tags
+release = "0.1.0dev0"
+# The short X.Y version
+version = "0.1.0dev0"
```

Comment:
If you want to single-source the version information, you could try something like:

```py
from importlib import metadata


release = version = metadata.version("watertap-seto")
```